### ISSUE TITLE
Statistics on the bases

### ISSUE THREAD

**Issue by kgtodorov** [DOREMUS-ANR/knowledge-base#49]
@pasqLisena  Statistics shown in the readme files of the data bases often regard the number of files in the archive, and not the number of entities. E.g., itema3.item.tar.gz contains  2296 files, but much more entities (of type F22).

**Comment by rtroncy** [comment]
@kgtodorov You're talking about https://github.com/DOREMUS-ANR/knowledge-base/blob/master/data/itema3/README.md? Note that this file has NOT been updated recently, so you should not necessarily trust it. Furthermore, the latest dump in the repo has not been loaded in the endpoint so you will not get the same number if you count locally in the file and if you sparql query the endpoint.

Having said this, the column 'Num' is not really meaningful. What were you expecting? Not the number of files in the archive apparently, but a count of entities? This will be a different entity for each row.

**Comment by pasqLisena** [comment]
> 2296 files, but much more entities (of type F22).

True, but the main entity described in Itema3 is not F22 but F31 (Concert).
I am counting the F31 in this case.

> Having said this, the column 'Num' is not really meaningful. What were you expecting? Not the number of files in the archive apparently, but a count of entities? This will be a different entity for each row.

I am counting the main entity (i.e. F31)
